tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -20,11 +20,16 @@
 import './flags_webgpu';
 
 import {backend_util, DataStorage, DataType, engine, env, findBackend, KernelBackend, Rank, RecursiveArray, ShapeMap, slice_util, Tensor, Tensor2D, Tensor3D, Tensor4D, TimingInfo, util} from '@tensorflow/tfjs-core';
-import {Glslang} from '@webgpu/glslang/dist/web-devel/glslang.onefile';
 // TODO(xing.xu): use FusedConv2DConfig from backend_util:
 // https://github.com/tensorflow/tfjs/issues/2471
 // tslint:disable-next-line: no-imports-from-dist
 import {FusedConv2DConfig} from '@tensorflow/tfjs-core/dist/ops/fused_util';
+// TODO TODO to import computeOptimalWindowSize and other reduce_util function
+// from backend_util, rather than from 'dist'
+import {assertAxesAreInnerMostDims, computeOutAndReduceShapes} from '@tensorflow/tfjs-core/src/ops/axis_util';
```

Comment:
You can delete this import and the "TODO" comment above, and call `backend_util.assertAxesAreInnerMostDims` and `backend_util.computeOutAndReduceShapes` - they are already exported as part of `backend_util`.